web-api/src/Kerbero.Identity/Modules/Authentication/Controllers/AuthenticationController.cs

```diff
@@ -23,11 +21,7 @@ public AuthenticationController(IAuthenticationService authenticationService)
   [AllowAnonymous]
   public async Task<ActionResult> Login(LoginDto loginDto)
   {
-    var authenticatedDto = await _authenticationService.Login(loginDto);
-    await HttpContext.SignInAsync(CookieAuthenticationDefaults.AuthenticationScheme,
-      new ClaimsPrincipal(authenticatedDto.ClaimIdentity), authenticatedDto.Properties);
-
-    return new OkResult();
+    return Ok(await _authenticationService.Login(loginDto));
```

Comment:
```suggestion
    return await _authenticationService.Login(loginDto);
```
do not use Ok(payload) if you can return a type, it will be converted automatically, and the compiler can check the type